modules/pool/pool.service.ts

```diff
@@ -114,7 +114,7 @@ export class PoolService {
         return this.poolSwapService.getJoinExits(args);
     }
 
-    public async getFeaturedPoolGroups(): Promise<GqlPoolFeaturedPoolGroup[]> {
+    public async getFeaturedPoolGroups(chains: Chain[]): Promise<GqlPoolFeaturedPoolGroup[]> {
         const cached: GqlPoolFeaturedPoolGroup[] = await this.cache.get(
```

Comment:
@franzns I'm in doubt about how to re-write this and `poolGqlLoaderService.getFeaturedPoolGroups()` functions given `chains`. Should we try to get it from cache for every network and for the ones it doesn't find we get it from `poolGqlLoaderService`?